distributed/tests/test_worker_client.py

```diff
@@ -1,7 +1,10 @@
 from __future__ import print_function, division, absolute_import
```

Comment:
The worker_client testing file is for other features  (see http://distributed.readthedocs.io/en/latest/task-launch.html#submit-tasks-from-worker).  Putting these tests in their own `test_preload.py` file might be better.